app/hotels/src/navigation/HotelsNavigationOptions.js

```diff
@@ -84,7 +84,7 @@ export default ({
         <MapHeaderButton
           onPress={goToAllHotelsMap}
           icon={icon}
-          text={text}
+          text={text.id}
```

Comment:
Is that change a typo?

Reply:
No: 
```
type Props = {|
  +cityName?: string,
  +checkin?: string,
  +checkout?: string,
  +icon: React.Element<typeof Icon>,
  +text: {| +id: TranslationKeysType |},
  +goToAllHotelsMap: () => void,
|};
```

but it might be better to change this.

Reply:
@RobinCsl 🚀